package.json

```diff
@@ -58,6 +57,7 @@
     "trailingComma": "all"
   },
   "dependencies": {
+    "@jup-ag/core": "^1.0.0-beta.28",
```

Comment:
Is this package needed? I don't see it imported anywhere in this PR or in my local copy. 

I think it was added a long time ago and can now be deleted.

Reply:
Not sure how I ended up with 2 of the same comment. Resolving this one.